packages/docs/src/routes/community/layout!.tsx

```diff
@@ -1,22 +1,27 @@
 import { component$, Slot, useStyles$ } from '@builder.io/qwik';
+import {useLocation} from "@builder.io/qwik-city";
 import { SideBar } from '../../components/sidebar/sidebar';
 import { Footer } from '../../components/footer/footer';
 import { Header } from '../../components/header/header';
 import { OnThisPage } from '../../components/on-this-page/on-this-page';
 import { ContentNav } from '../../components/content-nav/content-nav';
 import type { RequestHandler } from '@builder.io/qwik-city';
-import { CommunityNavbar } from './components/community-navbar/community-navbar';
```

Comment:
pls leave that in place. guess this was just temp. removed or is smth not yet ready to ship 😄